source/common/quic/envoy_quic_client_stream.h

```diff
@@ -70,6 +70,9 @@ class EnvoyQuicClientStream : public quic::QuicSpdyClientStream,
   void OnTrailingHeadersComplete(bool fin, size_t frame_len,
                                  const quic::QuicHeaderList& header_list) override;
 
+  // Http::MultiplexedStreamImplBase
+  bool hasPendingData() override;
```

Comment:
should we disable flush timer for client streams as we do for HTTP/2? or maybe since we have separate classes only inherit for the server stream?

Reply:
It is disabled via overriding setFlushTimeout() to no-op a few lines above. 
hasPendingData() is implemented here just because it is pure method in the base class.

Reply:
ah, I missed it as it wasn't part of the diff, my bad!